docs/en/week07/07-3.md

```diff
@@ -0,0 +1,143 @@
+---
+lang-ref: ch.07-3
+title: Introduction to autoencoders
+lecturer: Alfredo Canziani
+authors: Vidit Bhargava, Monika Dagar
+date: 18 March 2021
+---
+$\newcommand{\vert}{\boldsymbol}$
```

Comment:
What is this???
`\vert` is the absolute value operator.
`\vect{}` is a shortcut for `\boldsymbol{}` already defined in the [preamble](https://github.com/Atcold/NYU-DLSP21/blob/master/docs/_layouts/custom.html#L7).